TxLib/Fluent/TxExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace Unclassified.TxLib.Fluent
+{
+    /// <summary>
+    ///     Provides fluent builder for Tx API
+    /// </summary>
+    public static class TxExtensions
+    {
+        #region Static members
+
+        /// <summary>
+        ///     Extension method for strings to create <see cref="TxFluentBuilder" />
+        /// </summary>
+        /// <param name="key">Text key to search.</param>
+        /// <returns>New TxFluentBuilder instance</returns>
+        public static TxFluentBuilder Tx(this string key)
+        {
+            return new TxFluentBuilder(key ?? string.Empty);
+        }
+
+        /// <summary>
+        ///     Extension method for strings to create <see cref="TxFluentBuilder" />. Ignores key case.
+        /// </summary>
+        /// <param name="key">Text key to search.</param>
+        /// <returns>New TxFluentBuilder instance</returns>
+        public static TxFluentBuilder Txi(this string key)
```

Comment:
Is this method really useful? It's not really case-insensitive anyway, it just converts the source text key to lower case and then goes into the (case-sensitive!) key search. I'm currently working on your suggested code (please don't update it) and would rename this method to `TxLower`.